### PR TITLE
Add remediations for mount options of removable partitions

### LINKED ISSUE #1881: needs remediation: xccdf_org.ssgproject.content_rule_mount_option_nodev_removable_partitions

### PR DESCRIPTION
**Pull request by jan-cerny**
Fixes #1881, #1882, #1883

### CODE REVIEW

File: shared/templates/static/bash/mount_option_noexec_removable_partitions.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# platform = Red Hat Enterprise Linux 7, multi_platform_fedora
+
+# Include source function library.
+. /usr/share/scap-security-guide/remediation_functions
+
+populate var_removable_partition
+
+NEW_OPT="noexec"
+
+if [ $(grep "$var_removable_partition" /etc/fstab | grep -c "$NEW_OPT" ) -eq 0 ]; then
```
Comment (shawndwells) [inline shared/templates/static/bash/mount_option_noexec_removable_partitions.sh]: If the remediation is being ran, we already know the mount option isn't there. Is the ``if`` check needed?

Comment (shawndwells) [inline shared/templates/static/bash/mount_option_noexec_removable_partitions.sh]: hmm. suppose there are people that dump the remediation scripts, so these need to be idempotent. nevermind.

File: shared/templates/static/bash/mount_option_nodev_removable_partitions.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# platform = Red Hat Enterprise Linux 7, multi_platform_fedora
```
Comment (shawndwells) [inline shared/templates/static/bash/mount_option_nodev_removable_partitions.sh]: This should be multi_platform_rhel

File: shared/templates/static/bash/mount_option_noexec_removable_partitions.sh (hunk 1 above)
Comment (jan-cerny) [inline shared/templates/static/bash/mount_option_noexec_removable_partitions.sh]: Yes, I think we usually don't rely on assumptions on what should be there.

File: shared/templates/static/bash/mount_option_nodev_removable_partitions.sh (hunk 2 above)
Comment (jan-cerny) [inline shared/templates/static/bash/mount_option_nodev_removable_partitions.sh]: OK, I will change this soon.

**Comment by jan-cerny** [comment]
I have made them multi_platform_rhel

**Comment by mpreisler** [comment]
ACK

**Pull request merged**